docs/pages/kubernetes-access/controls.mdx

```diff
@@ -72,9 +71,9 @@ and Teleport's roles in enterprise:
       name: admin
     spec:
       allow:
-        # if kubernetes integration is enabled, this setting configures which
+        # If kubernetes integration is enabled, this setting configures which
         # kubernetes groups the users of this role will be assigned to.
-        # note that you can refer to a SAML/OIDC trait named 'property' via the "{{external.property}}" template variable,
+        # Note that you can refer to a SAML/OIDC trait named 'property' via the "{{external.property}}" template variable,
```

Comment:
Can we remove this and just use the built in role `access`?

Reply:
In the EKS and Helm guides, I think most user roles are created using the `--roles` flag to keep things simple (and since we don't use SSO anywhere in them): https://goteleport.com/docs/kubernetes-access/helm/guides/custom/#step-4-create-a-teleport-user-optional

However, we support SSO for authenticating with kubernetes and allow this template field (`allow`) to be defined through interpolation to do so.

Reply:
I'm planning on going back through and checking several configuration files - also part of a general set of changes depending on https://github.com/gravitational/next/issues/275. We can verify, lint, and then move most of these command and config statements into the new component.